docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -216,8 +216,23 @@ The `status` section contains the requirements and attributes and has the follow
 | internalInvocationUrls | []string | A list of internal urls to invoke the function                                                    |
 | externalInvocationUrls | []string | A list of external urls to invoke the function, including ingresses and external-ip:function-port |
 
-<a id="status-example"></a>
+### Function state
+
+Here is a list of all possible function states:
+
```

Comment:
```suggestion
### Function state (`state`)

The `state` fields described the current function status, and can be one of the following:

```

Also - add the "Function State" link in the table of contents (sub-title to `status`)